src/main/kotlin/org/seekerwing/aws/sqsconsumer/MessageProcessor.kt

```diff
@@ -15,5 +15,17 @@ import software.amazon.awssdk.services.sqs.model.Message
  */
 interface MessageProcessor {
 
-    suspend fun processMessage(message: Message)
+    /**
+     * Processes a list of messages.
+     */
+    suspend fun processMessages(messages: Set<Message>): Set<Message> {
```

Comment:
can we document that the return object would contain a set of failed messages?

Reply:
good point. fixed it